addons/chartmuseum/resources/chartmuseum.cue

```diff
@@ -36,6 +36,7 @@ import (
 	}
 	DEBUG:              strconv.FormatBool(parameter.debug)
 	DISABLE_API:        strconv.FormatBool(parameter.disableAPI)
+	CACHE_INTERVAL:     strconv.FormatBool(parameter.cacheRefresh)
```

Comment:
I don't think FormatBool should be used here.

```suggestion
	CACHE_INTERVAL: parameter.cacheRefresh
```

Reply:
That's a mistake, was toying with disabling completely and used to be a bool.